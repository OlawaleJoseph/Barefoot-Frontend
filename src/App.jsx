import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import { Provider } from 'react-redux';
import App from './components/index';
import store from './store';
import 'bootstrap/dist/css/bootstrap.min.css';
import 'bootstrap/dist/js/bootstrap.bundle';

ReactDOM.render(
  <Provider store={store}>
    <Router>
      <Switch>
        <Route to="/" component={App} />
        <App />
      </Switch>
    </Router>
  </Provider>,
  document.getElementById('root'),
);
