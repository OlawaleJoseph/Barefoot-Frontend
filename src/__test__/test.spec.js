/* eslint-disable react/jsx-filename-extension */
/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import { shallow } from 'enzyme';
import React from 'react';
import App from '../components/index';

describe('Not Found', () => {
  it('Shoud display Not found', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.text()).toEqual('Welcome to BareFootNomad');
  });
});
