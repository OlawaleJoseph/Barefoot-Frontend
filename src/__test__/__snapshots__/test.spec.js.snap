// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Shoud display Not found 1`] = `ShallowWrapper {}`;
